cmd/minikube/cmd/config/prompt.go

```diff
@@ -36,7 +36,7 @@ func AskForYesNoConfirmation(s string, posResponses, negResponses []string) bool
 	reader := bufio.NewReader(os.Stdin)
 
 	for {
-		console.Out("%s [y/n]: ", s)
+		out.Out("%s [y/n]: ", s)
```

Comment:
Find a stutter-free replacement name for out.Out. Perhaps something likue `out.F` or `out.String`?

Reply:
yeah makes sense. I prefer String, since it's not a widely called function.